src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -185,138 +135,73 @@ public static Double Abs(Complex value)
                 return d * Math.Sqrt(1.0 + r * r);
             }
         }
+
         public static Complex Conjugate(Complex value)
         {
-            // Conjugate of a Complex number: the conjugate of x+i*y is x-i*y 
-
-            return (new Complex(value._real, (-value._imaginary)));
+            // Conjugate of a Complex number: the conjugate of x+i*y is x-i*y
+            return new Complex(value._real, -value._imaginary);
         }
+
         public static Complex Reciprocal(Complex value)
         {
             // Reciprocal of a Complex number : the reciprocal of x+i*y is 1/(x+i*y)
-            if ((value._real == 0) && (value._imaginary == 0))
+            if (value._real == 0 && value._imaginary == 0)
             {
-                return Complex.Zero;
+                return Zero;
             }
-
-            return Complex.One / value;
+            return One / value;
         }
-
-        // --------------SECTION: Comparison Operator(binary) Overloading -------------- //
-
+        
         public static bool operator ==(Complex left, Complex right)
         {
-            return ((left._real == right._real) && (left._imaginary == right._imaginary));
+            return left._real == right._real && left._imaginary == right._imaginary;
         }
+
         public static bool operator !=(Complex left, Complex right)
         {
-            return ((left._real != right._real) || (left._imaginary != right._imaginary));
+            return left._real != right._real || left._imaginary != right._imaginary;
         }
-
-        // --------------SECTION: Comparison operations (methods implementing IEquatable<ComplexNumber>,IComparable<ComplexNumber>) -------------- //
-
+        
         public override bool Equals(object obj)
         {
             if (!(obj is Complex)) return false;
-            return this == ((Complex)obj);
-        }
-        public bool Equals(Complex value)
-        {
-            return ((_real.Equals(value._real)) && (_imaginary.Equals(value._imaginary)));
+            return Equals((Complex)obj);
```

Comment:
@hughbe Did you do anything about this separately? This may not have been the place to fix a bug, but you'd still found and fixed a bug all the same; `((object)a).Equals((object)a)` should always be true.